spyder_kernels/customize/spyderpdb.py

```diff
@@ -34,6 +34,18 @@
 logger = logging.getLogger(__name__)
 
 
+def uses_comprehension(code):
+    """Check if given code uses comprehensions."""
+    comprehension_statements = (
+        ast.ListComp,
+        ast.SetComp,
+        ast.GeneratorExp,
+        ast.DictComp
+        )
```

Comment:
```suggestion
    )
```

Align with the line that opened the parenthesis to make this easier to read.